packages/hooks/src/useBreakpoints/useBreakpoints.test.ts

```diff
@@ -110,9 +110,9 @@ describe("useBreakpoints", () => {
     });
   });
 
-  describe("in between values", () => {
+  describe.each(["1", "2", "3", "4", "5", "6"])("in between values", () => {
     function randomBreakpoint(min: number, max: number) {
-      return Math.floor(min + Math.random() * (max - min + 1));
+      return Math.floor(min + Math.random() * (max - min));
```

Comment:
Before: `Math.floor(min + Math.random() * (max - min + 1))`  could be greater than the max breakpoint value causing the flakes.

If `Math.random()` was very close to 1 floating point multiplication can get weird
  * `min + 0.999... (max - min + 1)` 
  * ≈ <strike>min</strike> + .999... (max  - <strike>min</strike> + 1)
  *  0 ≈ `min - .999... * min` 
  * 0.999... * (max + 1) can be > max

After:
 *  `min + 0.999... (max - min)` 
 *  ≈ <strike>min</strike> + .999... (max  - <strike>min</strike>)
 * (0.999... * max) < max

Reply:
Based on the docs it seems to suggest it is an inclusive range where as we want an exclusive range
<img width="1372" alt="image" src="https://github.com/user-attachments/assets/59a7dc60-735a-44a9-abb1-13feff5415ae">

Reply:
The tests also seem to suggest that as well
https://github.com/lodash/lodash/blob/4.17.15/test/test.js#L18495-L18505

Reply:
I realized I should just subtract 1 from max and use lodash 🤦